### DISCUSSION TITLE
重启docker，配置丢失

### DISCUSSION THREAD

**Discussion by pengfei9403** [AlistGo/alist#1166] [Q&A]
通过docker desktop 安装的，配置成功后可以使用，但重启docker后就被重置了，不重启docker的情况下，删除container，再run一个新的，配置还在，什么情况？怎么保存配置？

**Comment by TreeHD** [comment]
新增一個volume
然後docker run的時候用 -v 掛載上去

**Comment by hululuuuuu** [comment]
估计是因为你们权限不足导致的在etc/alist 下创建文件失败
解决方案：可以在当前mac用户下新建文件夹 挂载 比如 /Users/xxx用户/alist/alistConfig:/opt/alist/data

**Reply by lovealan** [reply]
遇到了

添加了阿里对象存储, 使用是正常的, 重启docker后 提示 "failed get storage: storage not found; please add a storage first " , 管理后台 "存储" 功能列表无数据 .

看了挂载卷目录, 数据库 "data.db" 还在, "x_storages" 表 有阿里oss的记录数据, 但是页面上不会显示, 非常奇怪
